wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Details/ProfileDetailsContentController.swift

```diff
@@ -185,6 +189,10 @@ final class ProfileDetailsContentController: NSObject,
                 contents = [.readReceiptsStatus(enabled: readReceiptsEnabled)]
             }
 
+            if let conversation, Bundle.developerModeEnabled, !ProcessInfo.processInfo.isRunningTests {
+                contents.append(.messageProtocol(conversation.messageProtocol))
+            }
+
```

Comment:
**question**: why so many checks:
```
Bundle.developerModeEnabled, !ProcessInfo.processInfo.isRunningTests {
```
what are we trying to avoid?

Reply:
`Bundle.developerModeEnabled` ensures that we don't see this cell on production builds. `isRunningTests` ensures we don't see the cell in the snapshot tests.

Reply:
why don't we want to see it on prod? should this happen only when MLS is enabled both on client and server?

Reply:
It was never part of the design spec, but I needed a way to check whether a 1-1 was mls or proteus. It's just for dev verification.